src/sentry/static/sentry/app/views/eventsV2/breadcrumb.tsx

```diff
@@ -69,26 +73,34 @@ class DiscoverBreadcrumb extends React.Component<Props> {
     };
 
     return (
-      <PageHeading>
-        <CrumbLink to={target}>{t('Discover')}</CrumbLink>
-        <BetaTagWrapper>
-          <BetaTag />
-        </BetaTagWrapper>
+      <BreadcrumbList>
+        <BreadcrumbItem to={target}>{t('Discover')}</BreadcrumbItem>
         {this.getCrumbs()}
-      </PageHeading>
+        <BetaTag />
+      </BreadcrumbList>
     );
   }
 }
 
-const BetaTagWrapper = styled('span')`
-  margin-right: 0.4em;
+const BreadcrumbList = styled('span')`
+  display: flex;
+  align-items: center;
+  height: 40px;
 `;
 
-const CrumbLink = styled(Link)`
-  color: ${p => p.theme.gray2};
-  &:hover {
-    color: ${p => p.theme.gray2};
+const BreadcrumbItem = styled(Link)`
+  color: ${theme.gray2};
```

Comment:
We generally use `theme` from props instead of importing - it'll theoretically allow us to swap to different themes.

Reply:
Oh! Thanks